### PR TITLE
October news

### PR DESCRIPTION
**Pull request by maelle**
No new package as of now, and no blog post, which might change.

### CODE REVIEW

File: content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,594 @@
+---
+title: rOpenSci News Digest, October 2022
+author:
+  - The rOpenSci Team
+date: '2022-10-21'
+slug: ropensci-news-digest-october-2022
+categories: []
+tags:
+  - newsletter
+description: rOpenSci Champions programm call for champions and mentors; R-universe search; coworking
```
Comment (steffilazerte) [inline content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd:10]:
```suggestion
description: rOpenSci Champions Program call for champions and mentors; R-universe search; coworking
```

File: content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,594 @@
...
+Dear rOpenSci friends, it's time for our monthly news roundup!
+<!-- blabla -->
+You can read this post [on our blog](`r url`).
+Now let's dive into the activity at and around rOpenSci!
+
+## rOpenSci HQ
+
+### rOpenSci Champions Program: call for Champions and call for Mentors
+
+[rOpenSci Champions Program](/champions/) is part of a series of activities and projects we are carrying out to ensure our research software serves everyone in our communities.
```
Comment (steffilazerte) [inline content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd:46]:
```suggestion
The [rOpenSci Champions Program](/champions/) is part of a series of activities and projects we are carrying out to ensure our research software serves everyone in our communities. 
```

File: content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,594 @@
...
+### rOpenSci Champions Program: call for Champions and call for Mentors
+
+[rOpenSci Champions Program](/champions/) is part of a series of activities and projects we are carrying out to ensure our research software serves everyone in our communities. 
+That is why this program focuses on __people who belong to groups that are historically and systematically excluded__ from the open software and research software communities and who are interested in contributing to rOpenSci and the broader open source and research software communities.
+
+#### Call for Champions
+
+If you are selected as a Champion, over the course of a year you will develop a project, get training, and be mentored. You will also receive a small stipend. 
+
+You can apply until **November 7, 2022** [using a form](/champions/#how-to-apply).  You can read more details in this [blog post](/blog/2022/09/22/launch-champions-program/).
```
Comment (steffilazerte) [inline content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd:53]:
```suggestion
You can apply until **November 7, 2022** with this [form](/champions/#how-to-apply).  You can read more details in this [blog post](/blog/2022/09/22/launch-champions-program/).
```

File: content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd [hunk 4]
```diff
@@ -0,0 +1,594 @@
...
+
+### Coworking sessions continue!
+
+Join us for social coworking & office hours monthly on first Tuesdays! 
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+* [Tuesday, 01 November 2022 9 AM North American Pacific / 16:00 UTC ](/events/coworking-2022-11/) "Champions Program Application Clinic" *Hosted by rOpenSci community manager [Yanina Bellini Saibene](/author/yanina-bellini-saibene/) and [Steffi LaZerte](/author/steffi-lazerte/)*
```
Comment (steffilazerte) [inline content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd:69]:
```suggestion
* [Tuesday, November 1st 9 AM North American Pacific / 16:00 UTC ](/events/coworking-2022-11/) "Champions Program Application Clinic" *Hosted by rOpenSci community manager [Yanina Bellini Saibene](/author/yanina-bellini-saibene/) and [Steffi LaZerte](/author/steffi-lazerte/)*
```

File: content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd [hunk 5]
```diff
@@ -0,0 +1,594 @@
...
+    * Cowork on a project of your choice;
+    * Prepare your application for the Champions Program;
+    * Or talk to [Yanina](/author/yanina-bellini-saibene/) about the application process.
```
Comment (steffilazerte) [inline content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd:72]:
```suggestion
    * Prepare your application for the Champions Program.
    * Talk to [Yanina](/author/yanina-bellini-saibene/) about the application process.
    * Cowork independently on work related to R. Work on packages that tend to be neglected; What ever you need to get done!
```

File: content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd [hunk 6]
```diff
@@ -0,0 +1,594 @@
...
+    
+* [Tuesday, December 6th, 9 AM Australian Western / 1:00 UTC](/events/coworking-2022-12/) "Getting started with targets!" *Hosted by community host [Nick Tierney](/author/nicholas-tierney/) and [Steffi LaZerte](/author/steffi-lazerte/)*
+    - Dive into the world of targets! Do some studying; Start setting it up for some of your projects.
+    - Cowork independently on work related to R. Work on packages that tend to be neglected; What ever you need to get done!
```
Comment (steffilazerte) [inline content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd:76]:
```suggestion
    - Dive into the world of targets! Do some studying; Start setting it up for some of your projects;
    - Ask [Nick](/author/nicholas-tierney/) for suggestions how how to get started or tips and tricks;
    - Cowork independently on work related to R. Work on packages that tend to be neglected; What ever you need to get done!
```

File: content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd [hunk 7]
```diff
@@ -0,0 +1,594 @@
...
+
+### Are you ready to develop packages?
+
+If the question applies to _you_ as an R user, please refer to the resources listed in our [dev guide](https://devguide.ropensci.org/building.html#learning-about-package-development).
+Now, in [Susan Johnston’s wise words](https://github.com/susjoh/fibonacci), if you can open R, if you can install a package, if you can write functions or learn how to write functions, you can write an R package!
+
+If the question applies to _your machine_, you might like:
+
+* [`devtools::dev_sitrep()`](https://devtools.r-lib.org/reference/dev_sitrep.html);
+* [`devtools::has_devel()`](https://rdrr.io/cran/pkgbuild/man/has_compiler.html);
```
Comment (steffilazerte) [inline content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd:568]:
```suggestion
* [`devtools::has_devel()`](https://r-lib.github.io/pkgbuild/reference/has_compiler.html);
```

File: content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd [hunk 8]
```diff
@@ -0,0 +1,594 @@
...
+As the maintainer of an rOpenSci package, feel free to contact us on Slack or email `info@ropensci.org` to get your call for maintainer featured in the next newsletter. -->
+
+## Package development corner
+
+Some useful tips for R package developers. :eyes:
+
+### Are you ready to develop packages?
+
+If the question applies to _you_ as an R user, please refer to the resources listed in our [dev guide](https://devguide.ropensci.org/building.html#learning-about-package-development).
+Now, in [Susan Johnston’s wise words](https://github.com/susjoh/fibonacci), if you can open R, if you can install a package, if you can write functions or learn how to write functions, you can write an R package!
```
Comment (steffilazerte) [inline content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd:563]:
```suggestion
If the question applies to _you_ as an R user, know that in [Susan Johnston’s wise words](https://github.com/susjoh/fibonacci), if you can open R, if you can install a package, if you can write functions or learn how to write functions, you can write an R package! If you're still not sure please refer to the resources listed in our [dev guide](https://devguide.ropensci.org/building.html#learning-about-package-development).
```

File: content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd [hunk 9]
```diff
@@ -0,0 +1,594 @@
...
+
+If the question applies to _your machine_, you might like:
```
Comment (steffilazerte) [inline content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd:565]:
```suggestion
On the other hand, if this question applies to _your machine_, you might use the following functions to find out:
```

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle, looks good! Just a couple of wording suggestions. 

However, I wasn't able to get Netlify to deploy the preview, so if that gets fixed, I'll take another look at the "official" version (here I just perused the Rmd). 

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

**Comment by maelle** [comment]
Thanks a lot @steffilazerte! Here's the link to the preview I had to re-launch too :sweat_smile: https://deploy-preview-405--ropensci.netlify.app/

File: content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd [hunk 10]
```diff
@@ -0,0 +1,606 @@
...
+
+If you are selected as a Champion, over the course of a year you will develop a project, get training, and be mentored. You will also receive a small stipend. 
+
+You can apply until **November 7, 2022** with this [form](/champions/#how-to-apply).  You can read more details in this [blog post](/blog/2022/09/22/launch-champions-program/).
+
+#### Call for Mentors
+
+### Introducing R-universe power search
+
+Earlier this year we announced [r-universe search](https://ropensci.org/blog/2022/03/23/runiverse-search/) which lets you easily browse and discover R packages across the various R ecosystems. We have now upgraded the search system to support advanced search queries, to search for particular properties of an R package.
```
Comment (steffilazerte) [inline content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd:59]: Just for consistency 😅 
```suggestion
Earlier this year we announced [R-universe search](https://ropensci.org/blog/2022/03/23/runiverse-search/) which lets you easily browse and discover R packages across the various R ecosystems. We have now upgraded the search system to support advanced search queries, to search for particular properties of an R package.
```

**Comment by steffilazerte** [comment]
Looks good!

**Review by yabellini — COMMENTED** [review]
TExt for the Community Call

File: content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd [hunk 11]
```diff
@@ -0,0 +1,606 @@
...
+{{< tweet user="opencpu" id="1573334441849667584" >}}
+
+Advanced search lets you query packages that specifially match a particular author, maintainer, keyword, or topic. But you can also filter based on metadata derived by the build system, for example packages that contain a particular function or dataset, or packages with a particular contributor or dependency.
+
+To give it a try, simply go to https://r-universe.dev and click the advanced search icon:
+
+
+[![screenshot of the advanced search box](advanced.png)](https://r-universe.dev/search)
+
+
```
[the remark this answers is not in the document]
Comment (maelle) [reply content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd:70]: added!

File: content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd [hunk 12]
```diff
@@ -0,0 +1,608 @@
...
+That is why this program focuses on __people who belong to groups that are historically and systematically excluded__ from the open software and research software communities and who are interested in contributing to rOpenSci and the broader open source and research software communities.
+
+#### Call for Champions
+
+If you are selected as a Champion, over the course of a year you will develop a project, get training, and be mentored. You will also receive a small stipend. 
+
+You can apply until **November 7, 2022** with this [form](/champions/#how-to-apply).  You can read more details in this [blog post](/blog/2022/09/22/launch-champions-program/).
+
+#### Call for Mentors
+
```
Comment (yabellini) [inline content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd:56]:
```suggestion
We are seeking **mentors** to support our inaugural cohort of rOpenSci Champions.

Mentors will play a vital role in the rOpenSci Champions Program by helping select the first cohort of Champions and advising and inspiring their mentees. Mentors will also __connect__ their mentees to people, programs, and organizations; __recommend__, resources, readings, training, and experiences; and provide __feedback__ on the mentee's project.
 
[Learn the details](/blog/2022/10/20/mentors-champions-program/) and [express your interest](https://airtable.com/shrlrLdQbb4wphosK) by __Monday November 7, 2022__.

If you have any questions, please, contact our Community Manager by [email](mailto:yabellini@ropensci.org) or [book a 15 minutes meeting with her](https://calendly.com/yabellini-ropensci/15min).

We will have a [Community Call on October 24th](/commcalls/oct2022-champions/)to explain and explore the benefits of Champions Programs and a [Coworking slot on November 1st](/events/coworking-2022-11/) to answer any questions you might have about mentoring in this program.
```

Comment (yabellini) [reply content/blog/2022-10-21-ropensci-news-digest-october-2022/index.Rmd:56]: The blog post is merged but is not shown on the webpage.

**Review by yabellini — COMMENTED** [review]
Adding call for mentors. Attention: the blog post was merged with an ok build but is not shown on the webpage.  I build one more time on my local machine, which is working.  Perhaps it is something about the featured: true tag?

**Pull request merged**